cmd/neofs-adm/docs/subnetwork-usage.md

```diff
@@ -81,17 +81,17 @@ $ neofs-adm morph subnet client add \
 Add client request sent successfully.
 ```
 
-**NOTE:** owner of the subnetwork is also allowed to add clients. This is
+**NOTE:** the owner of the subnetwork is also allowed to add clients. This is
 the only one command that accepts `ownerID`, not the public key.
-Administrator can manage only his group (a group where that administrator
-has been added by subnet owner).
+Administrator can manage only its group (a group where that administrator
```

Comment:
`their`?

Reply:
Personally, i would use `their`, but it's not quite common in our repos. But, if you supports this option, I would change it :)